x-pack/plugins/security_solution/public/common/components/event_details/cti_details/threat_details_view.tsx

```diff
@@ -0,0 +1,176 @@
+import { DEFAULT_INDICATOR_SOURCE_PATH } from '../../../../../common/constants';
+import { CtiEnrichment } from '../../../../../common/search_strategy/security_solution/cti';
+import {
+  getShimmedIndicatorValue,
+  getEnrichmentValue,
+  getFirstElement,
+  isInvestigationTimeEnrichment,
+} from './helpers';
+import * as i18n from './translations';
+import { EnrichmentIcon } from './enrichment_icon';
+import { QUERY_ID } from '../../../containers/cti/event_enrichment/use_investigation_enrichment';
+import { InspectButton } from '../../inspect';
+
+const getFirstSeen = (enrichment: CtiEnrichment): number => {
+  const firstSeenValue = getShimmedIndicatorValue(enrichment, FIRSTSEEN);
+  const firstSeenDate = Date.parse(firstSeenValue ?? 'no date');
+  return Number.isInteger(firstSeenDate) ? firstSeenDate : new Date(-1).valueOf();
+};
+
+const ThreatDetailsHeader: React.FC<{
+  field: string | undefined;
+  value: string | undefined;
+  provider: string | undefined;
+  type: string | undefined;
+}> = ({ field, value, provider, type }) => (
+  <>
+    <EuiTextColor color="subdued">
+      <EuiFlexGroup justifyContent="flexStart" alignItems="center" gutterSize="s">
+        <EuiFlexItem grow={false}>
+          <EnrichmentIcon type={type} />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiText size="s">
+            {field} {value}
+          </EuiText>
+        </EuiFlexItem>
+        {provider && (
+          <>
+            <EuiFlexItem grow={false}>
+              <EuiText size="s">
+                {i18n.PROVIDER_PREPOSITION} {provider}
+              </EuiText>
+            </EuiFlexItem>
+          </>
+        )}
+        <EuiFlexItem>
+          <EuiHorizontalRule margin="none" />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    </EuiTextColor>
+    {isInvestigationTimeEnrichment(type) && (
+      <EuiFlexGroup justifyContent="flexEnd" gutterSize="none">
+        <EuiFlexItem grow={false}>
+          <InspectButton queryId={QUERY_ID} title={i18n.INVESTIGATION_QUERY_TITLE} />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    )}
+  </>
+);
+
+const ThreatDetailsDescription: React.FC<ThreatDetailsRow['description']> = ({
+  fieldName,
+  value,
+}) => {
+  const tooltipChild = [EVENT_URL, EVENT_REFERENCE].includes(fieldName) ? (
+    <EuiLink href={value} target="_blank">
+      {value}
+    </EuiLink>
+  ) : (
+    <span>{value}</span>
+  );
+  return (
+    <EuiToolTip
+      data-test-subj="message-tool-tip"
+      content={
+        <EuiFlexGroup direction="column" gutterSize="none">
+          <EuiFlexItem grow={false}>
+            <span>{fieldName}</span>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      }
+    >
+      {tooltipChild}
+    </EuiToolTip>
+  );
+};
+
+const columns: Array<EuiBasicTableColumn<SummaryRow>> = getSummaryColumns(ThreatDetailsDescription);
+
+const buildThreatDetailsItems = (enrichment: CtiEnrichment) =>
+  Object.keys(enrichment)
+    .sort()
```

Comment:
If I understand correctly, we were never sorting the fields on the details view. I did this to make those items more legible, since the fields response doesn't seem to enforce any kind of ordering and it was very hard to compare values across different indicators (ebd1b9d0f03860cdd71890c453840ec5b49a5a47). However, it is a subtle change that we should bring up with the team 👍